### ISSUE TITLE
Create API for retrieving Ordinals in user's account

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#3600]
So the developer doesn't need to getAddresses then query a server-side API just to get basic information about the Ordinals e.g. number.

**Comment by diegoxter** [comment]
Suscribing here, def need this

**Comment by markmhendrickson** [comment]
@diegoxter curious if this is a hard need on your end or more of a convenience method? In that you could query inscriptions per address separately from the wallet once the wallet provides the user's address

**Comment by diegoxter** [comment]
In my use case it would simplify the process to just query the user's address and get the inscriptions held, then work with that list later on. In any case if Hiro via the ordinals index provided this method (I think it does? Not sure about it) I don't see why it would be necessary to have it directly embedded in the wallet.

**Comment by markmhendrickson** [comment]
Makes sense! Here are the API docs for reference: https://docs.hiro.so/ordinals/
